Source/ACE/Entity/Landblock.cs

```diff
@@ -290,41 +365,49 @@ private void Broadcast(BroadcastEventArgs args, bool propogate, Quadrant quadran
                 players = this.worldObjects.Values.OfType<Player>().ToList();
             }
 
-            // filter to applicable players
-            players = players.Where(p => p.Location?.IsInQuadrant(quadrant) ?? false).ToList();
-
             switch (args.ActionType)
             {
                 case BroadcastAction.Delete:
                     {
-                        Parallel.ForEach(players, p => p.StopTrackingObject(wo));
+                        players = players.Where(p => p.Location?.IsInQuadrant(quadrant) ?? false).ToList();
+                        Parallel.ForEach(players, p => p.StopTrackingObject(wo, true));
                         break;
                     }
                 case BroadcastAction.AddOrUpdate:
                     {
+                        // supresss updating if player is out of range of world object.= being updated or created
+                        players = GetWorldObjectsInRange(wo, maxobjectRange, true).OfType<Player>().ToList();

```

Comment:
I am not so sure because keep in mid objects need to be marked as they come into range.   Player is just for players detected in range but objects also need to be detected